### PR TITLE
Documentation Site Updated

### PR DESCRIPTION
**Pull request by humzahasan**
- Added lazy loading for iframe
- Removed redundant code
- Added list componenty in CL
- Added grid system in CL

### CODE REVIEW

File: documentation/doc.css [hunk 1]
```diff
@@ -6,30 +6,43 @@
...
+.button-container,
+.typography-container,
+.list-container {
   padding: 2rem;
   margin: 2rem;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  border: 1px solid rgba(230, 149, 84, 0.329);
```
Comment (zoef-yt) [inline documentation/doc.css]: use var for colour

File: documentation/doc.css [hunk 2]
```diff
@@ -6,30 +6,43 @@
...
+  border-radius: 5px;
 }
 
 .image-avatar,
 .letter-avatar,
 .status-avatar {
-  margin: 1rem 0;
+  border: 1px solid rgba(230, 149, 84, 0.329);
```
Comment (zoef-yt) [inline documentation/doc.css]: use var for colour

File: documentation/doc.css [hunk 3]
```diff
@@ -6,30 +6,43 @@
...
 .badge-icon-section,
 .badge-avatar-section,
-.responsive-image-section {
-  margin: 1rem 0;
+.responsive-image-section,
+.typography-example-text,
+.typography-example-heading,
+.button-container-example,
+.normal-list-section {
+  border: 1px solid rgba(230, 149, 84, 0.329);
```
Comment (zoef-yt) [inline documentation/doc.css:50]: use var for colour

File: documentation/documentation.html [hunk 4]
```diff
@@ -36,71 +36,102 @@
...
+          onload="resizeIframe(this)"
+        ></iframe>
+        <iframe
+          loading="lazy"
+          id="typography-iframe"
+          src="./textstyle.html"
+          frameborder="0"
+          scrolling="no"
+          onload="resizeIframe(this)"
+        ></iframe> -->
```
Comment (zoef-yt) [inline documentation/documentation.html]: commented for future?

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: documentation/doc.css (hunk 1 above)
Comment (humzahasan) [reply documentation/doc.css]: Fixed.

File: documentation/doc.css (hunk 3 above)
Comment (humzahasan) [reply documentation/doc.css:50]: Fixed

File: documentation/documentation.html (hunk 4 above)
Comment (humzahasan) [reply documentation/documentation.html]: Yes

**Review by zoef-yt — APPROVED** [review]
Good commit message. Clean code. Nice variable names. Good job. Good to merge.

**Review by DivyanshSareen — APPROVED** [review]
Good to merge.

**Pull request merged**